doc/sphinx/practical-tools/coqide.rst

```diff
@@ -153,7 +153,7 @@ as standard |GtkSourceView| styles are available. Other styles can be
 added e.g. in ``$HOME/.local/share/gtksourceview-3.0/styles/`` (see
 the general documentation about |GtkSourceView| for the various
 possibilities). Note that the style of the rest of graphical part of
-Coqide is not under the control of |GtkSourceView| but of GTK+ and
+|CoqIDE| is not under the control of |GtkSourceView| but of GTK+ and
```

Comment:
`|GtkSourceView|` is correct?

Reply:
It's defined in this particular file as:

```
.. |GtkSourceView| replace:: :smallcaps:`GtkSourceView`
```